user-docs/modules/ROOT/pages/fdo-terminology.adoc

```diff
@@ -0,0 +1,43 @@
+= FIDO device onboarding terminology
+
+Learn more about the FIDO device onboarding terminology.
+
+.FDO terminology
+
+|===
+|Commands | Description
+
+| FDO |  FIDO Device Onboarding.
+| Device | Any hardware, device, or computer.
+| Owner | The final owner of the device - a company or an IT department.
+| Manufacturer | The device manufacturer.
+| Manufacturer server  | Creates the device credentials for the device.
+| Manufacturer client |  Informs the location of the manufacturing server.
+| Ownership Voucher (OV) | Record of ownership of an individual device.
+
+Contains the following information:
+
+* Owner (`fdo-owner-onboarding-service`)
+
+* Rendezvous Server - FIDO server (`fdo-rendezvous-server`)
+
+* Device (at least one combination)  (`fdo-manufacturing-service`)
+
+| Device Credential (DC) | Key credential and rendezvous stored in the device at manufacture.
```

Comment:
Should they be in alphabetical order? Also I think we're missing rendezvous definitions.

Reply:
Done